pkg/metrics/innerring.go

```diff
@@ -13,8 +13,8 @@ type InnerRingServiceMetrics struct {
 func NewInnerRingMetrics() InnerRingServiceMetrics {
 	var (
 		epoch = prometheus.NewGauge(prometheus.GaugeOpts{
-			Namespace: namespace,
-			Subsystem: innerRingSubsystem,
+			Namespace: innerRingNameSpace,
+			Subsystem: stateSubsystem,
```

Comment:
Whoa. Not that easy. That's an external node interface, we need to keep backwards compatibility or _all_ of the monitors/alerts will break with the new release.

Reply:
yes but:

1. we still tag out releases with as v**0**.X.Y;
2. we have `Updating from` sections for such things (at least we _had_ for such things);
3. we already did CLI args, config changes (`metrics` -> `prometheus`, `profile` -> `pprof`);
4. i have no explanations for a user why i give him such metrics:
```
# HELP neofs_node_object_epoch Current epoch as seen by inner-ring node.
# TYPE neofs_node_object_epoch gauge
neofs_node_object_epoch 3
```
(`object`)
```
# HELP neofs_node_object_epoch Current epoch as seen by inner-ring node.
# TYPE neofs_node_object_epoch gauge
neofs_node_object_epoch 6
```
(that is IR metrics but still `object` and also `neofs_node` (i remember no place where i can read "neofs node" and think about IR))

so i would classify that rather as a bug not a breaking change (commit that adds that const does not look like a decision, more like a mistake)

Reply:
1. Sure, but this doesn't really matter. When something is used, you can't easily break users assumptions about how it works unless you absolutely have to. NeoGo is 0.smth as well, but there is a https://github.com/nspcc-dev/neo-go/blob/master/ROADMAP.md#deprecated-functionality
2. Sure, we still do. And what I'd expect there is "please convert to this new smth before we drop it" and in some later version "btw, if you're still using that old smth, sorry, it's gone now".
3. And we've had a number of problems with these transitions with lots of broken configurations. infra-repo remembers a number of related commits fixing things months after the upgrade.
4. Yeah, that's a problem, but irrespective of how these metrics are called, people already use them. And they'd be somewhat disappointed to hear they need to change everything immediately after the upgrade.

It's OK to clean these things up, but give everyone some transition period. Most of the time it doesn't cost a lot, but people would be able to upgrade their networks with old configs/monitoring setups, ensure they work fine and then change whatever is needed to change. Otherwise it's too many things to handle at once during the upgrade.

Reply:
oh, so you were talking about deprecating them? i might get you wrong

Reply:
ok, i got you wrong: i thought that you did not want to remove anything. fixed.